docs/open_source/release-process.md

```diff
@@ -80,33 +115,22 @@ See Lerna issues [#1137](https://github.com/lerna/lerna/issues/1137) and [#1091]
 
 ## Publish to npm
 
-### For Minor Releases
+> TIP: You can use `npx lerna updated [--since=...]` to preview which packages it would publish before running the
+> commands in this section.
 
 ```
-$(npm bin)/lerna publish --skip-git --since=<previous-patch-tag>
+npx lerna publish --skip-git --since=<previous-tag>
 ```
 
-When lerna prompts for version, choose Minor.
+When lerna prompts for version, choose Major, Minor, or Patch as appropriate.
 
 Be sure to include the command-line flags:
 
 * `--skip-git` avoids immediately applying a git tag, which we will do later after updating the changelog
-* `--since=<previous-patch-tag>` (e.g. `--since=v0.36.1`) forces lerna to diff against the latest patch; otherwise,
-  it diffs against the previous minor release which may cause some packages without changes to be published (this
-  happens when bugfix releases aren't tagged from the master branch)
-
-### For Patch Releases
-
-```
-$(npm bin)/lerna publish --skip-git
-```
-
-When lerna prompts for version, choose Patch.
+* `--since=<previous-tag>` (e.g. `--since=v1.0.1`) forces lerna to diff against the latest tag; otherwise,
```

Comment:
it doesn't default to the latest tag? Or does it default to master?

Reply:
It normally finds the latest tag linearly back through git history, which will only be the latest actual tag in the case of x.0.0 being the latest release.